hypothesis-python/src/hypothesis/utils/conventions.py

```diff
@@ -27,5 +27,14 @@ def __repr__(self):
         return self.identifier
 
 
-infer = UniqueIdentifier(u'infer')
+class DefaultValueType(UniqueIdentifier):
+    # Using subclasses so they can be distinguished by e.g. Mypy
```

Comment:
nit: This comment should probably go at the top level as it applies to two classes, not just this one.